test/e2e/run_passwd_test.go

```diff
@@ -70,6 +70,11 @@ USER 1000`, ALPINE)
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 		Expect(session.OutputToString()).To(Not(ContainSubstring("passwd")))
+
+		// test that the /etc/passwd file is created
+		session = podmanTest.Podman([]string{"run", "--rm", "--user", "0:0", imgName, "ls", "/etc/passwd"})
```

Comment:
Why do we need --user to create it? Feels like it should work even when the default user is used, no?

Reply:
I think it only creates the user account if --user is specified and user is not listed in /etc/passwd.
I guess we could remove check for --user and always check if the default user is listed in the /etc/passwd.

Reply:
it is the current behavior.  I am fine with changing it but it is a separate change